datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -427,33 +433,15 @@ public GmsGraphQLEngine(
         this.dataPlatformInstanceType = new DataPlatformInstanceType(entityClient);
         this.accessTokenMetadataType = new AccessTokenMetadataType(entityClient);
         this.testType = new TestType(entityClient);
+        this.dataHubPolicyType = new DataHubPolicyType(entityClient);
+        this.dataHubRoleType = new DataHubRoleType(entityClient);
         // Init Lists
-        this.entityTypes = ImmutableList.of(
-            datasetType,
-            corpUserType,
-            corpGroupType,
-            dataPlatformType,
-            chartType,
-            dashboardType,
-            tagType,
-            mlModelType,
-            mlModelGroupType,
-            mlFeatureType,
-            mlFeatureTableType,
-            mlPrimaryKeyType,
-            dataFlowType,
-            dataJobType,
-            glossaryTermType,
-            glossaryNodeType,
-            containerType,
-            notebookType,
-            domainType,
-            assertionType,
-            versionedDatasetType,
-            dataPlatformInstanceType,
-            accessTokenMetadataType,
-            testType
-        );
+        this.entityTypes =
+            ImmutableList.of(datasetType, corpUserType, corpGroupType, dataPlatformType, chartType, dashboardType,
```

Comment:
Really hate this type of formatting change.

Reply:
Makes the code much less readable

Reply:
Agreed, sorry it's just the autoformatter. Reverted and added my changes using old format